strawberryfields/backends/__init__.py

```diff
@@ -76,10 +76,11 @@
     shared_ops
 """
 
-from .base import BaseBackend, BaseFock, BaseGaussian, ModeMap
+from .base import BaseBackend, BaseFock, BaseGaussian, BaseBosonic, ModeMap
```

Comment:
The module docstring above will have to be changed so that the new bosonic features are included in the documentation. This is important because Sphinx directly uses those references to create the documentation.

It is also worth checking that e.g., the new ``BaseBosonic`` class appears in the list of backends, the docstring of functions render correctly, etc. For this, the `docs/readthedocs.org:strawberryfields` CI check is really helpful, as it offers a "new version" of the SF documentation - so worth checking out that all's fine there!

Reply:
Will modify